crawl-ref/source/mon-ench.cc

```diff
@@ -326,6 +326,12 @@ void monster::add_enchantment_effect(const mon_enchant &ench, bool quiet)
              name(DESC_THE).c_str());
         break;
 
+    case ENCH_BORGNJORS_VILE_CLUTCH:
```

Comment:
This is currently being printed separately for each monster in the area of effect, right? How hard would it be to combine these messages? "Zombie hands grab the quokka, the juggernaut, and Cerebov" seems better than spending three whole lines on it, but if it's not easy to do I don't think it's a huge deal.

Reply:
It would be nice, but probably some significant work to accumulate those messages. I'm not sure if we have any good facility for combining messages over a beam explosions, unless the targets all have the same name, in which case the message system already does the message combination since they are identical.